client/extensions/zoninator/components/settings/zone/posts-list.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PostCard from './post-card';
+import RecentPostsDropdown from '../../recent-posts-dropdown';
+import SearchAutocomplete from './../../search-autocomplete';
+import FormFieldset from 'components/forms/form-fieldset';
+import SortableList from 'components/forms/sortable-list';
+
+class PostsList extends Component {
+
+	addPost = ( { push } ) => {
+		return ( post ) => push( post );
+	}
+
+	removePost = ( { remove } ) => {
+		return ( idx ) => remove( idx );
```

Comment:
We should be consistent about using either `post` or `idx` in these methods tho, since they denote the same thing, don't they?

Reply:
They don't 🙂  `post` is an actual `post` while `idx` is the index in the list.